packages/react-native-bpk-component-carousel/src/BpkCarousel-test.common.js

```diff
@@ -42,7 +42,11 @@ const renderWithLayout = (
       layout,
     },
   };
-  rendered.toJSON().props.onLayout(mockNativeEvent);
+  const json = rendered.toJSON();
+  if (!json) {
+    throw new Error('');
```

Comment:
Can we make this error more descriptive?

Reply:
![oops](https://media1.tenor.com/images/363b51663ff60e0c40fe8e5e46200b06/tenor.gif?itemid=5026725)